src/lib/apirequest.ts

```diff
@@ -205,6 +209,8 @@ export function createAPIRequest<T>(
   // to 10MB.  Setting to 2GB by default.
   // https://github.com/google/google-api-nodejs-client/issues/991
   options.maxContentLength = options.maxContentLength || maxContentLength;
+  options.headers['Accept-Encoding'] = 'gzip';
```

Comment:
When this is specified, does axios automatically decompress the response?

Reply:
When it's specified, Google endpoints return a request with a `Content-Encoding` header set to `gzip`.  Axios will automatically decompress those requests:
https://github.com/axios/axios/blob/master/lib/adapters/http.js#L156

Reply:
Awesome.